### ISSUE TITLE
Set up a system for tracking the main themes of individual Open Data Day events

### ISSUE THREAD

**Issue by Daniel-Mietchen** [okfn/opendataday#149]
Paraphrased from [this email thread](https://groups.google.com/d/msg/open-data-day/ukgb74zaGcw/nLqQ4P_fEgAJ):

> We are currently not gathering information about the themes of Open Data Day events, but this doesn't mean we can't or shouldn't.

So to get things rolling, this ticket is meant to help us find out whether anyone is interested in such cross-linking and what suggestions there might be on how to go about implementation.

By way of an example, I am involved in organizing an event around the theme of open research data: https://github.com/sparcopen/Open-Research-doathon .

For events that likewise have a presence on GitHub, GitHub topics might provide a basic way of finding other Open Data Day projects, as per #148.

**Comment by tlacoyodefrijol** [comment]
@Daniel-Mietchen Do you mean something like getting tags from github like the [Civic Issue Finder](https://github.com/codeforamerica/civic-issue-finder) did?

**Comment by Daniel-Mietchen** [comment]
Yes, something like this. So far, the only dimensions we can work with are location and to some extent time. Both are important, but so is topic.

**Comment by tlacoyodefrijol** [comment]
Would tags and icons work? I'm just wondering how to avoid overbuilding the site but still be able to communicate topics. Also, maybe not every project will have a github repo but still be relevant to a certain community or group.
